JDI.Light/JDI.Light/Factories/WebDriverFactory.cs

```diff
@@ -242,15 +236,10 @@ public IWebDriver GetDriver(DriverType driverType)
 
         public string RegisterDriver(DriverType driverType)
         {
-            switch (RunType)
-            {
-                case RunType.Remote:
-                    return RegisterRemoteDriver(driverType);
-                default:
-                    return RegisterLocalDriver(driverType);
-            }
-
-            throw new Exception($"Can't register driver: {driverType}");
+            if (RunType == RunType.Remote)
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=5844553)